openedx/core/djangoapps/credit/models.py

```diff
@@ -22,6 +24,33 @@ class CreditCourse(models.Model):
     course_key = CourseKeyField(max_length=255, db_index=True, unique=True)
     enabled = models.BooleanField(default=False)
 
+    @classmethod
+    def is_credit_course(cls, course_key):
+        """ Check that given course is credit or not
+
+        Args:
+            course_key(CourseKey): The course identifier
+
+        Returns:
+            Bool True if the course is marked credit else False
+        """
+        return cls.objects.filter(course_key=course_key, enabled=True).exists()
+
+    @classmethod
+    def get_or_create_credit_course(cls, course_key):
+        """ Get the credit course if exists
+
+        Args:
+            course_key(CourseKey): The course identifier
+
+        Raises:
+            DoesNotExist if no CreditCourse exists for the given course key.
+
+        Returns:
+            CreditCourse if one exists for the given course key.
```

Comment:
Please change this text, since this `get_or_create` method will return a tuple `(object, created)`